lib/executor.js

```diff
@@ -171,10 +171,48 @@ function applyAppConfig(app, instructions) {
 
 function setupDataSources(app, instructions) {
   forEachKeyedObject(instructions.dataSources, function(key, obj) {
+    var opts = {
+      // do we need to limit dynamic-able keys? or everything can be dynamic
+      keys: ['host', 'url', 'port']
```

Comment:
Okay, then I can probably just add **env var** to the dynamic configuration implementation for middlewares, and reuse that :+1: